src/components/accounts/accounts.js

```diff
@@ -114,6 +146,7 @@ export class AccountsComponent {
     this.initMoreBtns();
     this.addEventListeners();
     this.initAddAccountDialogComponent();
+    this.initConfirmDialog();
```

Comment:
`this.initAddAccountDialogComponent();` and `this.initConfirmDialog();` have comon name of method `mountChildren`